proto/warden/async/v1beta1/metrics.proto

```
@@ -0,0 +1,75 @@
+// under the Apache 2.0 license governing the rest of the project.
+// Refer to the LICENSE file in this module's directory for full terms.
+// Use, modification, and distribution are strictly limited.
+// Do NOT use this file unless you agree to the terms stated in that license.
+//
+// SPDX-FileCopyrightText: 2025 Warden Labs
+// SPDX-License-Identifier: LicenseRef-Proprietary-RestrictedModule
+
+syntax = "proto3";
+package warden.async.v1beta1;
+
+import "amino/amino.proto";
```

Buf (1.54.0)

import "amino/amino.proto": file does not exist

(COMPILE)